docker/nitro/Dockerfile

```diff
@@ -59,6 +59,12 @@ ENV \
     AARCH64_UNKNOWN_LINUX_MUSL_OPENSSL_INCLUDE_DIR=/work/veracruz-nitro-root-enclave/musl/include \
     X86_64_UNKNOWN_LINUX_MUSL_OPENSSL_INCLUDE_DIR=/work/veracruz-nitro-root-enclave/musl/include
 
+RUN wget https://musl.cc/${ARCH}-linux-musl-native.tgz && \
+    tar zxvf ${ARCH}-linux-musl-native.tgz && \
+    cp ${ARCH}-linux-musl-native/bin/${ARCH}-linux-musl-gcc /bin && \
+    cp ${ARCH}-linux-musl-native/bin/${ARCH}-linux-musl-g++ /bin && \
```

Comment:
Rather than copying the binaries, you can make a symbolic link, or better yet extend `$PATH`.